### ISSUE TITLE
Incomplete range in bruteforce_stdcrypto.py

### ISSUE THREAD

**Issue by ius** [santoku/Santoku-Linux#15]
In `bruteforce_stdcrypto.py` of Santoku 0.2, line 50-51:

``` python
# try all possible four digit PINs, leave the loop when found
for i in range(0,9999):
```

Which spans `[0, 9999)` instead of `[0, 9999]`.

A fix:

``` python
for i in range(10000):
```

**Comment by thomascannon** [comment]
Thanks for the fix, marked as a bug to address in 0.3. I think we should host the actual script and other tools on the public GitHub too so we can track the fixes.

**Comment by tfernandes** [comment]
Fixed in Alpha 0.3. 

Thanks!
